### PR TITLE
Chapter1/homework/1 and2

### PR DESCRIPTION
**Pull request by mihailfox**
changed numberOfPersons to constant; replaced tabs with spaces;

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            const int numberOfPersons = 2;
```
Comment (diverges) [inline Src/BootCamp.Chapter/Program.cs]: This is what a sample comment would look like in a PR.

**Comment by Alumniminium** [comment]
Next time, please give your pull request a proper title.

**Comment by mihailfox** [comment]
> Next time, please give your pull request a proper title.

Can you please give me an example for this particular PR ?
Cheers!

**Comment by Alumniminium** [comment]
Homework successfully submitted!

**Comment by Almantask** [comment]
Homework done.

**Pull request closed without merging**